keps/sig-apps/20200529-delete-priority-annotations

```diff
@@ -0,0 +1,293 @@
+---
```

Comment:
Please make this a .md file.

Reply:
Hi @janetkuo,
I've renamed the file. Please re-check. Thank you